tiles-generation/docker/vector-tile/Dockerfile

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+
+FROM amazonlinux:2
+
+RUN yum clean all && \
+    yum update -y && \
+    amazon-linux-extras install epel && \
+    yum install -y java-17-amazon-corretto-headless wget tar go make python37
+
+RUN wget https://github.com/onthegomap/planetiler/releases/download/v0.5.0/planetiler.jar
+
+RUN wget -O mbutil.tar.gz https://github.com/mapbox/mbutil/archive/refs/tags/v0.3.0.tar.gz && \
+    tar -xzf mbutil.tar.gz && \
+    cd /mbutil-0.3.0 && \
+    python3 setup.py install
+
+RUN wget -O s5cmd.tar.gz https://github.com/peak/s5cmd/archive/refs/tags/v2.0.0-beta.2.tar.gz && \
```

Comment:
Do we need to depend on beta version?

Reply:
Since 2.0 has some bug fixes and improvements, in our case it's stable enough to use the copy operation, and before 2.0 beta, the 1.4 was released last year Sep 21, 2021. https://github.com/peak/s5cmd/releases